### ISSUE TITLE
AF: Group details should not look like links

### ISSUE THREAD

**Issue by ssin1901** [citizenos/citizenos-fe#1093]
**What is the problem?**
@ilmartyrk @oksks @BeccaMelhuish this was always an issue but with the new Request to Add Topics to Group feature is going to become a problem.

The user is given to understand that all the links in the Global + Group Activity Feed work. However, clicking on the links will simply take them to the Group (see screenshot)

<img width="1439" alt="Screenshot 2024-04-09 at 13 48 08" src="https://github.com/citizenos/citizenos-fe/assets/72747107/f7c472e6-74b7-446c-91b7-4fe88f933891">

**Why is this a problem?**
1. Group admin may be under the impression that they will be able to "preview" the topic they are about to add
2. Group admin may be under the impression that clicking on the links will lead to the topic automatically be added to the group
3. Group admin may be under the impression that the user profile can be accessed via the link

**Possible solution.**
@ilmartyrk Becca suggested, as a quick fix to remove cursor:pointer from the links.

**Comment by BeccaMelhuish** [comment]
Yes just to add my suggestion for how this should work (though @kevincrepin can hopefully review it):  

It seems that in AF currently some of them are links, e.g. when I click anywhere in the block screenshot below it takes me to the Marketing meetings topic. But it should be the case that all names/titles other than the one that it's linked are black bold, and only the one that is a link is blue (topic title in this case below). 

And re the cursor, it should be that when hovering anywhere on the block that isn't the link, then it is a standard cursor. But when the user hovers over something that is a link (i.e. a blue bold title), only then does it turn to a hand cursor (so they know what to expect when they click, at the moment they don't know where clicking takes them).  

But, in relation to the 'Request to add' feature, we need to bear in mind when we revisit the UX for this (to make it intuitive, not needing instructions), that this AF isn't at all helpful in this case, as the link takes you to the group, and there is nothing in the group that appears telling you what to do (you have to know where the request is hiding, under the add button). 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/5f3a7ef6-527f-4f22-ac79-ad12ed79eb9f)

**Comment by ilmartyrk** [comment]
Will fix with complete activity feed overhaul it is a really complex issue that is related to the whole activity feed. Currently the whole activity is a link and there is no quick solution for that

**Comment by BeccaMelhuish** [comment]
@ilmartyrk @ssin1901 Added to AF review issue and moved to 'later'
